_plugins/sort_edtf.rb

```
@@ -1,10 +1,47 @@
+                         end
+
+        # Return a hash with numeric, original, and display_format
+        { numeric: numeric_value, original: str, display_format: display_format }
+      end
+
+      # Sort by the numeric representation
+      sorted_dates = parsed_dates.sort_by { |date| date[:numeric] }
+
+      # Return an array of "display_format:original" strings
+      sorted_dates.map { |date| "#{date[:display_format]}:#{date[:original]}" }
     end
```

rubocop

[convention] Assignment Branch Condition size for sort_edtf is too high. [<13, 20, 10> 25.87/23]

(Metrics/AbcSize)